data/phi2331/phi027/__cts__.xml

```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi2331.phi027" xml:lang="lat" groupUrn="urn:cts:latinLit:phi2331">
             <ti:title xml:lang="lat">Tacitus</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi2331.phi027.perseus-lat2" workUrn="urn:cts:latinLit:phi2331.phi027">
-              <ti:label xml:lang="eng">Tacitus, The Scriptores historiae augustae Volume III</ti:label>
-              <ti:description xml:lang="eng">Vopiscus, Flavius, fl. 3./4. Jh, attributed author; Magie, David, b. 1877, editor</ti:description>
+              <ti:label xml:lang="lat">Tacitus</ti:label>
```

Comment:
Is there a different designation for the emperor Tacitus versus the historian? Just curious.